src/API/Configurations.js

```diff
@@ -0,0 +1,90 @@
+import { useQueryClient } from 'react-query';
+
+import { useOkapiQuery } from './useOkapiQuery';
+import { useOkapiMutation } from './useOkapiMutation';
+
+
+export const useListQuery = options => {
+  const query = useOkapiQuery({
+    path: 'remote-storage/configurations',
+    queryKey: 'remote-storage/configurations',
+    ...options,
+  });
+
+  return {
+    configurations: query.data?.configurations ?? [],
+    ...query,
+  };
+};
+
+
+export const useSingleQuery = ({ id, onError, ...rest }) => {
+  const queryClient = useQueryClient();
+
+  const query = useOkapiQuery({
+    path: `remote-storage/configurations/${id}`,
+    queryKey: ['remote-storage/configurations', id],
```

Comment:
I believe `remote-storage/configurations` key can be made as a constant

Reply:
thanks for pointing out, refactored it